bin/deploy

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+require('dotenv').config()
+const { exec } = require("child_process");
+
+var type = process.env.DEPLOY_TYPE || "cloudfront"
+
+var keys = [
+  "CachePolicyID",
+  "CachePriceClass",
+  "CacheDomainName",
+  "CacheSSLCertificate",
+  "SourceBucket",
+  "IiifLambdaMemory",
+  "IiifLambdaTimeout",
+  "OriginRequestARN",
+  "OriginResponseARN",
+  "PixelDensity",
+  "ResolverTemplate",
+  "ViewerRequestARN",
+  "ViewerRequestType",
+  "ViewerResponseARN",
+  "ViewerResponseType"
+]
+
+let parameter_overrides = ""
+keys.forEach(element => {
+  if(process.env[element] !== undefined) {
+    parameter_overrides = parameter_overrides + ` ${element}="${process.env[element]}"`
+  }
+})
+
+var cmd = `cd sam/${type} && sam deploy --capabilities CAPABILITY_IAM CAPABILITY_AUTO_EXPAND --region ${process.env['AWS_REGION']} --profile ${process.env['AWS_PROFILE']} --stack-name ${process.env['AWS_STACK_NAME']} --resolve-s3 --parameter-overrides ${parameter_overrides}`
+console.log(cmd)
+
+exec(cmd, (error, stdout, stderr) => {
```

Comment:
this is some grade a Node.js bs. they dont have a process replacing exec at all.